gql/builder.go

```diff
@@ -281,7 +281,7 @@ func (ob *ObjectBuilder) buildFields(sType reflect.Type, parent string, baseFiel
 				},
 				sortArgumentName: &graphql.ArgumentConfig{
 					Description: "Sort the list, ie `{Field: \"position\", Order: \"ASC\"}`",
-					Type:        graphqlListFilter,
+					Type:        graphqlSortFilter,
```

Comment:
why didn't a test catch this issue?

Reply:
Because basically it only affects the schema name and description, the parsing of the list filter and sort filter are both done the same way